app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +92,63 @@ class Identification extends React.Component {
       });
   }
 
+  getLanguageOptions(languageType) {
```

Comment:
This method only ever needs to be called once, right? Because the supported languages aren't going to change when filling out this form, so I would think you could call this method once and store it in the local state. This would make `getLanguageValue()` faster as well. I'm not actually sure what including the `type` field in these options is doing either.

Reply:
the type field is for primary vs secondary language.  I needed it for the onLanguageChange event, but I refactored to have that passed in as a parameter instead